diff --git a/hakyll.cabal b/hakyll.cabal
index fcded8d..ac532a8 100644
--- a/hakyll.cabal
+++ b/hakyll.cabal
@@ -198,11 +198,11 @@ Library
 
   If flag(previewServer)
     Build-depends:
-      wai             >= 3.2   && < 3.3,
-      warp            >= 3.2   && < 3.3,
-      wai-app-static  >= 3.1   && < 3.2,
-      http-types      >= 0.9   && < 0.13,
-      fsnotify        >= 0.2   && < 0.4
+      wai,
+      warp,
+      wai-app-static,
+      http-types,
+      fsnotify
     Cpp-options:
       -DPREVIEW_SERVER
     Other-modules:
